src/main/java/org/broadinstitute/hellbender/engine/filters/AmbiguousBaseReadFilter.java

```diff
@@ -7,38 +7,42 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
 /**
- * Filters out reads with more than a threshold number of N's
+ * Filters out reads with more than a threshold number of N's. By default will determine the threshold by the given
+ * maximum fraction of bases in the read. This can be overridden by setting the maximum number of ambiguous bases instead.
  */
 @DocumentedFeature(groupName= HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY)
 public final class AmbiguousBaseReadFilter extends ReadFilter {
 
     private static final long serialVersionUID = 1L;
+    private final Integer maxAmbiguousBases;
```

Comment:
Agreed that `--ambigFilterBases` is a better name for keep consitency. Thanks for taking into account my comments and suggestions!